### ISSUE TITLE
PW doesn't work at all. 

### ISSUE THREAD

**Issue by Cooldude9210** [sschmid/pw#3]
1. Installed via the bash -c "$(curl -fsSL https://raw.githubusercontent.com/sschmid/pw/main/install)" command, got this output:

> Cloning into '/usr/local/opt/pw'...
>remote: Enumerating objects: 409, done.
>remote: Counting objects: 100% (33/33), done.
>remote: Compressing objects: 100% (22/22), done.
>remote: Total 409 (delta 14), reused 21 (delta 8), pack-reused 376
>Receiving objects: 100% (409/409), 115.22 KiB | 2.06 MiB/s, done.
>Resolving deltas: 100% (202/202), done.
>Linking /usr/local/opt/pw/src/pw to /usr/local/bin/pw
>ln: /usr/local/bin/pw: File exists
2. Installed fzf, got this output:
> /opt/homebrew/Cellar/fzf/0.34.0: 17 files, 3.0MB`
3. Running anything, even "pw help" results in:
>/usr/local/bin/pw: line 14: declare: -g: invalid option
>declare: usage: declare [-afFirtx] [-p] [name[=value] ...]`

Not sure how to proceed.

**Comment by sschmid** [comment]
@Cooldude9210 
Could you please check with bash version you have installed?

```
echo $BASH_VERSION
```

If you're default shell is zsh, do this:
```
bash
echo $BASH_VERSION
```

You need bash-5.x, macOS default bash version is very old (bash-3.2)

You can install latest bash like this
```
brew install bash
```

**Comment by sschmid** [comment]
I released [pw-4.5.1](https://github.com/sschmid/pw/releases/tag/4.5.1) which now checks if you have at lease `bash-4.2` installed.
